docs/src/main/asciidoc/spring-integration-support.adoc

```diff
@@ -33,6 +33,8 @@ Please refer to Javadoc for details.
 
 This starter provides the following 2 parts of configuration options:
 
+NOTE: From version 4.0.0, when the property of **spring.cloud.azure.eventhubs.processor.checkpointStore.createContainerIfNotExists** is not enabled manually, no Storage container will be created automatically with the name from **spring.cloud.azure.eventhubs.eventHubName**.
```

Comment:
We shoud use uniform format for these properties, can we change this to kebab here?